### ISSUE TITLE
Tests fail with 'relative import path' error

### ISSUE THREAD

**Issue by cdoremus** [uki00a/fresh-testing-library#57]
I'm putting together a PR for the Fresh repo on examples I created for my recent blog post on `fresh-testing-library`. Here is the PR: https://github.com/denoland/fresh/pull/1990 and here is the branch I created the PR from: https://github.com/cdoremus/fresh/tree/fresh-testing-lib .

When I run the tests I created using the Deno task I created for them (`deno task test:www`), I come up with the following error that points to the `fresh-testing-library` code: 
```
craig@craig-G750JX:~/git/deno/denoland/fresh/fresh_fork$ deno task test:www
Task test:www deno test -A tests/www/
error: Relative import path "preact-render-to-string" not prefixed with / or ./ or ../ and not in import map from "https://deno.land/x/fresh_testing_library@0.11.0/internal/fresh/mod.ts"
```

I am using the latest  FTL version (0.11.0) and my branch was just merged with the latest code from the Fresh `main` branch .

Please advise.

**Comment by uki00a** [comment]
Hi @cdoremus, thanks for reporting the problem! This issue should be fixed in [0.11.1](https://github.com/uki00a/fresh-testing-library/releases/tag/0.11.1).

**Comment by cdoremus** [comment]
> Hi @cdoremus, thanks for reporting the problem! This issue should be fixed in [0.11.1](https://github.com/uki00a/fresh-testing-library/releases/tag/0.11.1).

Thank you , Yuki! My tests now work and my PR now passes the Fresh repo's CI pipeline.
